tests/docker/Dockerfile.install

```diff
@@ -37,20 +38,23 @@ RUN apt-get update && \
     strace \
     gdb \
     openssh-server \
-    psmisc
+    psmisc \
+    vim \
+    less
 
 COPY . /comdb2
 
 RUN cd /comdb2 &&  \
     rm -rf build && \
     mkdir build && \
     cd build && \
-    cmake -DWITH_TCL=1 .. && \
-    make -j4 && \
-    make -j4 test-tools && \
-    make install &&  \
+    cmake -DWITH_TCL=1 -DCMAKE_BUILD_TYPE=Debug -G Ninja .. && \
+    time ninja && \
```

Comment:
I have a tendency to add the Debug build type here as well.  It would be nice if there were a way to do this without having to hard-wire it into this Docker file.

Reply:
We can have a separate Dockerfile for debug builds (maybe FROM a container that has all the prerequisites installed).